compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/ts-non-null-expression-default-value.tsx

```diff
@@ -0,0 +1,8 @@
+export const Component = ({theme = localStorage.getItem('theme')!}) => {
```

Comment:
I know it's just an example, but localStorage is a mutable data source that isn't safe to read from in a component.

Reply:
Thanks @josephsavona. I ammended my commit to remove the `localStorage` usage from the fixture. I was struggling to come up with a contrived example.

Reply:
awesome thanks!